pom.xml

```diff
@@ -116,7 +116,7 @@
       <dependency>
         <groupId>com.squareup</groupId>
         <artifactId>javapoet</artifactId>
-        <version>1.0.0</version>
+        <version>1.3.0</version>
```

Comment:
Unrelated change (I suppose, unless it's required by the few code changes above)